packages/CollabCodeEdit-Core.package/CCEMessage.class/class/forwardEditMessage..st

```diff
@@ -1,4 +1,4 @@
 server-Message creation
-forwardEditMessage: aSerializeddCCEEdit
+forwardEditMessage: aCCEEdit
 	^ self new: (CCEMessage serverMsgTypes at: #receiveEdit:) 
```

Comment:
Please rename serverMsgTypes to a squeak-style name, perhaps serverMessageTypes?

Reply:
2nd